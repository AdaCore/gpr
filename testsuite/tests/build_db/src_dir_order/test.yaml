description:
    Test for source resolution based on the order of the source_dirs
    declarations
driver: build_and_run
project_file: test
main: main
