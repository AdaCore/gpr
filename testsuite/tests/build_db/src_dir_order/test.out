/tree/src2/pkg_a.ads
/tree/src3/pkg_b.ads
